Code/GraphMol/CIPLabeler/CIPMol.h

```diff
@@ -0,0 +1,99 @@
+//
+//
+//  Copyright (C) 2020 Schrödinger, LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#pragma once
+
+#include <memory>
+
+#include <GraphMol/RDKitBase.h>
+
+#include "Descriptor.hpp"
+#include "Mancude.h"
+
+namespace RDKit {
+
+namespace CIPLabeler {
+
+template <typename T, typename U> class CIPMolIterator {
+public:
+  class CIPMolIter {
+  private:
+    ROMol *mol;
```

Comment:
It's not necessary to follow the convention, but I do find that labelling data members makes code much easier to read.
So I'd name these members: 
```
ROMol *dp_mol;
U d_pos;
T d_current;
```

Reply:
obviously this comment applies to all class members.
Again, not necessary to make this change if you don't agree that it makes reading code easier.

Reply:
Yeah, it totally makes sense. I realized about the data members once I had ported most of the code, but deferred doing something about it until I had some tests running. I guess now is a good moment for it.